site/docs/v1/tech/guides/user-actions.adoc

```diff
@@ -0,0 +1,238 @@
+---
+layout: doc
+title: User Actions
+description: Using user actions
+navcategory: developer
+---
+:page-liquid:
+
```

Comment:
Please add a link to this guide to the API pages:

https://fusionauth.io/docs/v1/tech/apis/actioning-users

and 

https://fusionauth.io/docs/v1/tech/apis/user-actions